cypress/e2e/banUser.cy.js

```diff
@@ -0,0 +1,34 @@
+describe('Ban User Form', () => {
```

Comment:
Can you also write a test to message a user, I made on from the driver side https://github.com/IgorSteps/easypark-ui/blob/main/cypress/e2e/messageAdmin.cy.js